### PR TITLE
fix(arc-saas): detailed list of tenant onboarding

### LINKED ISSUE #76: (arc-saas): onboarding Tenant list page should be more detailed with plan details also
**Is your feature request related to a problem? Please describe.**
onboarding Tenant list page should be more detailed like we can add more details in that like plan details ,contact details etc

### PR DESCRIPTION
**Pull request by Deepika516**
## Description
1.) adding list of tenant with of the details like subscription plan , plan details etc

Fixes #76 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/saas-ui/src/app/shared/models/tenantDetails.model.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+export class tenantDetails {
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/shared/models/tenantDetails.model.ts:1]: class name

File: projects/saas-ui/src/app/main/components/tenant-detail/tenant-detail.component.spec.ts
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/tenant-detail/tenant-detail.component.spec.ts:1]: test cases missing

File: projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts [hunk 2]
```diff
@@ -47,103 +42,136 @@ export class OnboardingTenantListComponent extends RouteComponentBaseDirective {
...
+  getTenantDetails() {
+    this.tenantFacade.getTenantDetails().subscribe(resp => {
+      console.log(resp);
+      this.rowData = resp.map(item => {
+        if (item) {
+          const fullTenantName = [
+            item.contacts[0]?.firstName,
+            '    ',
+            item.contacts[0]?.lastName,
+          ]
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts:144]: what if he is created via lead

File: projects/saas-ui/src/app/main/components/tenant-detail/tenant-detail.component.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        tenantList => {
+          const tenantData = tenantList.find(
+            tenant => tenant.id === this.tenantId,
+          );
+          if (tenantData) {
+            const fullTenantName = [
+              tenantData.contacts[0]?.firstName,
+              '    ',
+              tenantData.contacts[0]?.lastName,
+            ]
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/tenant-detail/tenant-detail.component.ts:49]: why are we getting entire list and not just single tenant using tenantId

File: projects/saas-ui/src/app/shared/models/tenantDetails.model.ts (hunk 1 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/shared/models/tenantDetails.model.ts:1]: Done

File: projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts (hunk 2 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts]: done Sorted

File: projects/saas-ui/src/app/main/components/tenant-detail/tenant-detail.component.spec.ts
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/tenant-detail/tenant-detail.component.spec.ts:1]: Done

**Pull request merged**